app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -299,6 +301,19 @@ public boolean isHealthCodeExportEnabled() {
     public void setHealthCodeExportEnabled(boolean enabled) {
         this.healthCodeExportEnabled = enabled;
     }
+    
+    /** {@inheritDoc} */
+    @Override
+    public Long getMinSupportedVersion() {
+    	return minSupportedVersion;
+    }
+    
+    /** {@inheritDoc} */
+    @Override
+    public void setMinSupportedVersion(Long version) {
+    	Long maxAllowedVersion = this.version != null ? this.version : 0L;
+    	this.minSupportedVersion = version <= maxAllowedVersion ? version : maxAllowedVersion;
```

Comment:
I'm not sure we want to enforce this. It's logical, but it's not necessary. They can go back or get rid of a version limit if they want.